BCStudentSoftwareDevTeam/celts
# Fix Footer Positioning

Summary:
This PR fixes issue #34 
The text in the footer was breaking for small screen sizes. It was also not occupying the whole width of big-screen sizes. To fix this, we added some CSS stylings and made sure that the text does not break anymore, while also ensuring the footer sticks to the bottom of big-screen sizes.

To test:
Go to any web page and inspect the page. Choose different small screen sizes and you should see that the text never breaks now. Also, when the web page content increases, the footer also responds to these changes by staying at the bottom of the page.
